src/components/themed/TransactionRow.js

```diff
@@ -185,6 +185,9 @@ const getStyles = cacheStyles((theme: Theme) => ({
     height: theme.rem(1.75),
     borderRadius: theme.rem(0.5)
   },
+  iconArrows: {
```

Comment:
This does not make any sense. How does adding a margin only to the left center the icon?